src/Symfony/Component/Notifier/Bridge/Slack/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+7.1
+---
+
+* Add `SlackButtonBlockElement` to add button as accessory to a section block
+* Add `emoji` and `verbatim` options to `text` and `field` methods in `SlackSectionBlock`
```

Comment:
```suggestion
7.2
---

 * Add `SlackButtonBlockElement` to add button as accessory to a section block
 * Add `emoji` and `verbatim` options to `text` and `field` methods in `SlackSectionBlock`
```

/.2 because we re in feature freeze for 7.1